elements/pfe-band/src/pfe-band.json

```diff
@@ -84,7 +84,6 @@
           "type": "string",
           "enum": [
             "lightest",
-            "lighter",
```

Comment:
It seems like a breaking change to do away with the `lighter` option.  Perhaps we could just assign it the same #fff color for now and add documentation to that effect?

Reply:
Per our discussion in scrum, I left this out (recommended from design team) but I did bring it back to the demo files for card & band to show it still works; it defaults to base color, which is quite similar to the original "lighter" color.